github/src/main/kotlin/gropius/sync/github/Outgoing.kt

```diff
@@ -184,6 +186,82 @@ class Outgoing(
         }
     }
 
+    /**
+     * Mutate an AddedLabelEvent upto GitHub
+     * @param imsProjectConfig active config
+     * @param issueInfo info of the issue containing the timeline item
+     * @param userList users that have contributed to the event
```

Comment:
missing doc for label

Reply:
f4cea4b1d0191373ae90c82a78fcdbe78e5cf47d